velox/exec/fuzzer/PrestoQueryRunner.cpp

```diff
@@ -291,7 +307,8 @@ std::optional<std::string> PrestoQueryRunner::toSql(
     for (auto i = 0; i < aggregates.size(); ++i) {
       appendComma(i, sql);
       const auto& aggregate = aggregates[i];
-      sql << toCallSql(aggregate.call);
+      sql << toCallSql(
+          aggregate.call, aggregate.sortingKeys, aggregate.sortingOrders);
```

Comment:
perhaps, add a check that aggregate.distinct is false; otherwise, we may end up silently generating invalid SQL